packages/ui/src/theme/css/component/textField.scss

```diff
@@ -1,5 +1,4 @@
 .amplify-textfield {
-  flex-direction: column;
```

Comment:
This moved to Field styles, is that what I'm gathering?

Reply:
that is correct